internal/commands/app/create_inputs.go

```diff
@@ -82,9 +82,9 @@ func (i *createInputs) Resolve(profile *user.Profile, ui terminal.UI) error {
 	return nil
 }
 
-func (i *createInputs) resolveName(ui terminal.UI, client realm.Client, r appRemote) error {
+func (i *createInputs) resolveName(ui terminal.UI, client realm.Client, filter realm.AppFilter) error {
```

Comment:
lets change this function signature to just take `groupID, appID string` arguments, and you can construct the `realm.AppFilter` inside this function